storage/remote/otlptranslator/prometheusremotewrite/combined_appender_test.go

```diff
@@ -713,6 +643,106 @@ func TestCombinedAppenderSeriesRefs(t *testing.T) {
 		err := capp.AppendHistogram(ls, Metadata{}, 4, 2, nil, nil)
 		require.Error(t, err)
 	})
+
+	t.Run("metadata is NOT updated when appendMetadata flag is false", func(t *testing.T) {
```

Comment:
nit: it would be more explicit if this and the next test was a loop with `_, appendMetadata := range([]bool{true, false})`. So that the input is guaranteed to be the same between the two related tests.